src/Routes/SamplePage/SamplePage.js

```diff
@@ -82,23 +86,36 @@ const SamplePage = () => {
         <PageHeaderTitle title="Sample Provisioning App" />
         <p> Provisioning Demo </p>
       </PageHeader>
-      <Main>
-        <Stack hasGutter>
-          {images && <StackItem>{renderSelect(images)}</StackItem>}
-          <StackItem>
-            <FormGroup label="Image AMI">
-              <TextInput id="ami" value={chosenImage.id || ''} onChange={onInputChange} />
-            </FormGroup>
-          </StackItem>
-          <StackItem>
-            <Title headingLevel="h2" size="3xl"></Title>
-            <Button variant="primary" onClick={() => setWizardModal(true)}>
-              Open Wizard
-            </Button>
-            {isWizardOpen && <ProvisioningWizard isOpen={true} onClose={() => setWizardModal(false)} image={chosenImage} />}
-          </StackItem>
-        </Stack>
-      </Main>
+      <Grid hasGutter>
+        {images && <GridItem>{renderSelect(images)}</GridItem>}
+        <GridItem span={3}>
+          <FormGroup label="Provider">
+            <Select
+              variant={SelectVariant.single}
+              aria-label="Select Provider"
```

Comment:
Can you add this as well? :) 
```suggestion
              aria-label="Select Provider"
              ouiaId="select_provider"
```